test/default_manifold.jl

```diff
@@ -753,8 +752,8 @@ Base.size(x::MatrixVectorTransport) = (size(x.m, 2),)
         @test X3 == log(M, p, q)
         @test log!(M, X3, p, q) == log(M, p, q)
         @test X3 == log(M, p, q)
-        @test inverse_retract(M, p, q, CustomDefinedInverseRetraction()) == -2 * Y
-        @test distance(M, p, q, CustomDefinedInverseRetraction()) == 2.0
+        @test inverse_retract(M, p, q, CustomDefinedInverseRetraction()) == -Y
+        @test distance(M, p, q, CustomDefinedInverseRetraction()) == 1.0
```

Comment:
I think you should maybe modify `CustomDefinedInverseRetraction` to keep the old test the same?

Reply:
took me a while to see that I probably implemented the mutating variant wrongly, but I got it back from the allocating one I deleted, so sure, returning to the modified form is better for sure.